### PR TITLE
[ID] Added missing translation lines

### PR DESCRIPTION
**Pull request by elzdave**

### CODE REVIEW

File: locales/id/id.json [hunk 1]
```diff
@@ -433,15 +433,15 @@
...
+    "Please confirm your password before continuing.": "Harap konfirmasikan kata sandi Anda sebelum melanjutkan.",
     "Please copy your new API token. For your security, it won't be shown again.": "Harap salin token API baru Anda. Untuk keamanan, token tidak akan ditampilkan lagi.",
     "Please enter your password to confirm you would like to log out of your other browser sessions across all of your devices.": "Harap isi kata sandi anda untuk mengkonfirmasi bahwa anda ingin keluar dari sesi browser lain yang ada di semua perangkat milik Anda.",
     "Please provide a maximum of three receipt emails addresses.": "Harap berikan maksimal tiga alamat surel tanda terima.",
     "Please provide the email address of the person you would like to add to this team.": "Harap isi alamat surel orang yang ingin anda tambahkan ke tim ini.",
     "Please provide your name.": "Harap isi nama anda.",
     "Poland": "Polandia",
     "Portugal": "Portugal",
-    "Press \/ to search": "Tekan \/ untuk melakukan pencarian",
+    "Press / to search": "Tekan / untuk melakukan pencarian",
```
Comment (andrey-helldar) [inline locales/id/id.json:444]: You do not need to remove character escapes.

File: locales/id/id.json [hunk 2]
```diff
@@ -538,7 +538,7 @@
     "Sri Lanka": "Sri Lanka",
     "Standalone Actions": "Aksi Mandiri",
     "Start Polling": "Mulai Jajak pendapat",
-    "State \/ County": "Negara Bagian \/ Kabupaten",
+    "State / County": "Negara Bagian / Kabupaten",
```
Comment (andrey-helldar) [inline locales/id/id.json:541]: You do not need to remove character escapes.

**Review by andrey-helldar — CHANGES_REQUESTED** [review]

**Comment by andrey-helldar** [comment]
Why do you want to remove escaping?

**Review by andrey-helldar — APPROVED** [review]

**Comment by andrey-helldar** [comment]
Thank you

**Comment by andrey-helldar** [comment]
@elzdave, can you translate the last key?

https://laravel-lang.com/statuses/id.html

**Comment by elzdave** [comment]
@andrey-helldar uh my bad, that line is skipped 😂😂
Okay, I'll make one PR to fix it

**Pull request merged**